luskan/src/main/java/org/opencadc/luskan/impl/LocalServiceURI.java

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
This class is not necessary and a separate config does not appear to be necessary. The serviceURI key=value should be in TempStorageManager.properties. So move lines 104-105 into TempStorageManager and delete this file.